website/docs/r/code_engine_app.html.markdown

```diff
@@ -0,0 +1,195 @@
+* `scale_min_instances` - (Optional, Integer) Optional minimum number of instances for this app. If you set this value to `0`, the app will scale down to zero, if not hit by any request for some time.
+* `scale_request_timeout` - (Optional, Integer) Optional amount of time in seconds that is allowed for a running app to respond to a request.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your resource is created.
+
+* `id` - The unique identifier of the code_engine_app.
+* `app_id` - (String) The identifier of the resource.
+  * Constraints: The maximum length is `36` characters. The minimum length is `36` characters. The value must match regular expression `/^[0-9a-z]{8}-[0-9a-z]{4}-[0-9a-z]{4}-[0-9a-z]{4}-[0-9a-z]{12}$/`.
+* `created_at` - (String) The date when the resource was created.
+* `endpoint` - (String) Optional URL to invoke app. Depending on visibility this is accessible publicly ot in the private network only. Empty in case 'managed_domain_mappings' is set to 'local'.
+* `endpoint_internal` - (String) URL to app that is only visible within the project.
+* `entity_tag` - (String) The version of the app instance, which is used to achieve optimistic locking.
+  * Constraints: The maximum length is `63` characters. The minimum length is `1` character. The value must match regular expression `/^[\\*\\-a-z0-9]+$/`.
+* `href` - (String) When you provision a new app,  a URL is created identifying the location of the instance.
+  * Constraints: The maximum length is `2048` characters. The minimum length is `0` characters. The value must match regular expression `/(([^:\/?#]+):)?(\/\/([^\/?#]*))?([^?#]*)(\\?([^#]*))?(#(.*))?$/`.
+* `id` - (String) The identifier of the resource.
+  * Constraints: The maximum length is `36` characters. The minimum length is `36` characters. The value must match regular expression `/^[0-9a-z]{8}-[0-9a-z]{4}-[0-9a-z]{4}-[0-9a-z]{4}-[0-9a-z]{12}$/`.
+* `resource_type` - (String) The type of the app.
+  * Constraints: Allowable values are: `app_v2`.
+* `status` - (String) The current status of the app.
+  * Constraints: Possible values are: `ready`, `deploying`, `failed`, `warning`.
+* `status_details` - (List) The detailed status of the application.
+Nested scheme for **status_details**:
+	* `latest_created_revision` - (String) Latest app revision that has been created.
+	* `latest_ready_revision` - (String) Latest app revision that reached a ready state.
+	* `reason` - (String) Optional information to provide more context in case of a 'failed' or 'warning' status.
+	  * Constraints: Allowable values are: `ready`, `deploying`, `waiting_for_resources`, `no_revision_ready`, `ready_but_latest_revision_failed`.
+* `etag` - ETag identifier for code_engine_app.
+
+## Provider Configuration
```

Comment:
You can remove 
`# Provider Configuration`
`### Static credentials`
`### Environment variables` sections from all code engine docs as it is generic doc and is available in provider page - https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs#authentication

Reply:
Removed the paragraphs